examples/resources/bloxone_ipam_address_block/resource.tf

```diff
@@ -0,0 +1,24 @@
+resource "b1ddi_ip_space" "example" {
+  name = "example_ip_space"
+  tags = {
+    location = "site1"
+  }
+}
+
+resource "b1ddi_address_block" "example" {
+  address = "192.168.1.0"
+  cidr    = 24
+  name    = "example_address_block"
+  space   = b1ddi_ip_space.example.id
+}
+
+resource "b1ddi_address_block" "example" {
```

Comment:
shouldn't put same name in the same tf file, if at all someone decides to copy this as is, it will fail for them. 

Also it would be best to separate optional fields so that it is easy to understand to the user. Eg. 
https://github.com/infobloxopen/terraform-provider-bloxone/blob/35d0e1d2ace37835ce215edd00b8371606f41499/examples/resources/bloxone_ipam_subnet/resource.tf#L12-L23